compile.sh

```diff
@@ -31,6 +31,12 @@ FORCE_CHECKOUT="yes"			# ignore manual changes to source
 BUILD_ALL="no"				# cycle through available boards and make images or kernel/u-boot packages.
 					# set KERNEL_ONLY to "yes" or "no" to build all packages/all images
 
+PRIVATE_CCACHE="no"			# Use separate ccache directory located in build tree ($DEST/ccache)
+					# Can be used to avoid problems with ownershipt on $HOME/.ccache. 
+					# Set to "yes" if you are using build script as non-root user with "sudo"
+					# note: to use a ramdisk for ccache, create first a symlink to $DEST/ccache
+                                        # Set to "no" check if $DEST/ccache exists and delete (directory/symlink)
```

Comment:
... and yes, it will delete $DEST/ccache if it exists when PRIVATE_CCACHE="no" to switch back to $HOME/.ccache

In my opinion, if a tool creates mess (root owned files in $HOME/.ccache) it is better to offer the option to deal/avoid the mess by design (and not by luck/goodwill/etc.) so having a separate folder for ccache is a good option.